client/containers/Pub/PubHeader/DraftReleaseButtons.js

```diff
@@ -136,7 +105,7 @@ const DraftReleaseButtons = (props) => {
 			typeof latestRelease.sourceBranchKey !== 'number' ||
 			latestRelease.sourceBranchKey < latestKey;
 		return (
-			<>
+			<React.Fragment>
```

Comment:
for my own knowledge: is this a stylistic choice, or is there a substantive difference between the empty el and the fragment?

Reply:
Stylistic at the end of the day. But also the babel syntax highlighter for SublimeText doesn't recognize `<>` as a tag and causes all sorts of highlighting problems.